docs/examples.md

```diff
@@ -1,39 +1,41 @@
 # Examples
 
-To get you started, there are some example packages included that you can use. You can take a look at the code on Github or you
-can use the CLI included to generate a new package using these examples. Here is how you can do that:
+To get you started, there are some example packages included that you can use. You can [take a look at the code on Github](https://github.com/amzn/style-dictionary/tree/master/example/) or you can use the CLI included to generate a new package using some of these examples. Here is how you can do that:
+
 ```bash
-$ mkdir MyStyleD
-$ cd MyStyleD
+$ mkdir MyFolder
+$ cd MyFolder
 $ style-dictionary init [example]
 ```
-Where `[example]` is one of: `basic`, `complete`, `npm`, `s3`
+
+Where `[example]` is one of: `basic`, `complete`.
```

Comment:
This has less to do with this specific PR, but I want to get my thoughts out somewhere and it is sort of related. I've been thinking about in the future, maybe 3.0, having more of a scaffolding CLI through npx where the user could answer questions about their project and we would generate a starter package for them. Kind of like Yeoman. Here is an article outlining that: https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b